package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "soniak-bew-front",
+  "version": "1.0.0",
+  "description": "",
+  "main": "dist/app.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "npx tsc && cp -R views dist && node dist/app.js",
+    "build": "npx tsc && cp -R views dist"
```

Comment:
Start command could be.

`npm run build && node dist/app.js`

This would mean if you need to change the build you would only need to change one script not two